Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -31,21 +31,20 @@ struct PlaceDataSourceError: Error {
 
 class PlaceCarouselViewController: UIViewController {
 
-    fileprivate let minimumTimeAtLocationBeforeFetchingEvents: TimeInterval = 15 * 60
     fileprivate let currentLocationIdentifier = "CURRENT_LOCATION"
     fileprivate let MIN_SECS_BETWEEN_LOCATION_UPDATES: TimeInterval = 1
 
-    private let timeOfLastLocationUpdateKey = "timeOfLastLocationUpdate"
-
     fileprivate var timeOfLastLocationUpdate: Date? {
```

Comment:
Nope, this is still used as we still set `timeOfLastLocationUpdate` key in this class (for as long as it is before I move the location handling out of `PlaceCarouselViewController`